pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -229,6 +229,10 @@ func (r *Replica) RangeFeed(
 		if err != nil {
 			return future.MakeCompletedErrorFuture(err)
 		}
+
+		// Notify rangefeed restart limiter about catch up scan.
+		catchupDone := r.store.feedRestarter.startCatchup()
```

Comment:
That would only partially help us as we can't track updates to limiter as it could be updated before or after setting notification. But reading its value makes good sense. No separate tracking.